pkg/setting/setting_jwt.go

```diff
@@ -26,17 +26,15 @@ type AuthJWTSettings struct {
 }
 
 type ExtJWTSettings struct {
-	Enabled        bool
-	ExpectIssuer   string
-	ExpectAudience string
```

Comment:
I propose we nix this config parameter before we have any clients of it. Also, there isn't a real use case for being able to specify this differently for the many Grafana instances.

Reply:
Yes it is fine to remove that one :+1: